src/btree/row_modify.c

```diff
@@ -365,6 +366,7 @@ __wt_update_obsolete_check(
      * Only updates with globally visible, self-contained data can terminate update chains.
      */
     for (first = NULL, count = 0; upd != NULL; upd = upd->next, count++) {
+        WT_MEMORY_PREFETCH(upd->next);
```

Comment:
Can this be used in the search flow?

Reply:
Yes, it can used in `txn_read_...` functions. I will create another ticket to do those changes.